modules/account-lib/src/coin/avaxc/walletInitializationBuilder.ts

```diff
@@ -0,0 +1,60 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics/dist/src/base';
+import { BuildTransactionError, NotImplementedError } from '../baseCoin/errors';
+import { TransactionBuilder, DEFAULT_M } from './transactionBuilder';
+import { Transaction } from './transaction';
+
+export class WalletInitializationBuilder extends TransactionBuilder {
+  private _owners: string[] = [];
+
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+  }
+
+  // region Base Builder
+  /** @inheritdoc */
+  protected async buildImplementation(): Promise<Transaction> {
+    throw new NotImplementedError('buildImplementation not implemented');
+  }
+
+  /** @inheritdoc */
+  initBuilder(tx: Transaction): void {
+    throw new NotImplementedError('initBuilder not implemented');
+  }
+
+  // endregion
+
+  // region Common builder methods
+  /**
+   * Set one of the owners of the multisig wallet.
+   *
+   * @param {string} address The public key of the owner's account
```

Comment:
I think the owners will be set as addresses and not as public keys, because that is what the contract expects to.
```suggestion
   * @param {string} address The address of the owner's account
```